data/mods/Aftershock/maps/mapgen_pallete/houses.json

```
@@ -0,0 +1,96 @@
+[
+  {
+    "type": "palette",
+    "//": "the food should be rotted, but for now the regular food item groups can work for now",
```

```suggestion
    "//": "the food should be rotted, but the regular food item groups can work for now",
```
Yes, I'm compulsive...

---

lol